test/__tests__/data/salesforce_input.json

```diff
@@ -793,7 +1168,45 @@
       "timestamp": "2020-01-27T17:50:57.109+05:30",
       "type": "identify",
       "userId": "1e7673da-9473-49c6-97f7-da848ecafa76"
-    }
+    },
+    "mockNetworkResponses": [
+      {
+        "type": "post",
```

Comment:
I believe we can also include `type` into `reqParams`. So, the input becomes simple.

```
[
  {
    request: { ... },
    response: { ... }
  },
  {
    request: { ... },
    response: { ... }
  },
  ...
]
```